Source/Blazorise/Base/BaseAfterRenderComponent.cs

```diff
@@ -83,6 +98,11 @@ protected virtual void Dispose( bool disposing )
         /// </summary>
         protected bool Disposed { get; private set; }

```

Comment:
Is this property still needed?

Reply:
Yes, because there are still some components that are disposing synchronously.